content/desktop/install/rhel.md

```diff
@@ -0,0 +1,136 @@
+---
+description: Instructions for installing Docker Desktop on RHEL
+keywords: red hat, red hat enterprise linux, rhel, rpm,
+  update install, uninstall, upgrade, update, linux,
+  desktop, docker desktop, docker desktop for linux, dd4l
+title: Install Docker Desktop on RHEL
+toc_max: 4
+download-url-base: https://download.docker.com/linux/rhel
+---
+
+> **Docker Desktop terms**
+>
+> Commercial use of Docker Desktop in larger enterprises (more than 250
+> employees OR more than $10 million USD in annual revenue) requires a [paid
+> subscription](https://www.docker.com/pricing/).
+
+This page contains information on how to install, launch and upgrade Docker Desktop on a Red Hat Enterprise Linux (RHEL) distribution.
+
+{{< button text="RPM package" url="https://desktop.docker.com/linux/main/amd64/docker-desktop-4.28-x86_64-rhel.rpm?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64" >}}
+
+## Prerequisites
+
+To install Docker Desktop successfully, you must:
+
+- Meet the [system requirements](linux-install.md#general-system-requirements).
+- Have a 64-bit version of either RHEL 8 or RHEL 9.
+
+If you don't have `pass` installed, or it can't be installed, you must enable
+[CodeReady Linux Builder (CRB) repository](https://access.redhat.com/articles/4348511)
+and
+[Extra Packages for Enterprise Linux (EPEL)](https://docs.fedoraproject.org/en-US/epel/).
+
+   {{< tabs >}}
```

Comment:
tip: if you add the sam group to these tab groups will switch synchronously

```suggestion
   {{< tabs group="os_version" >}}
```

Reply:
Nice!